kong/plugins/oauth2/access.lua

```diff
@@ -605,7 +607,8 @@ local function retrieve_token(conf, access_token)
   local token, err
 
   if access_token then
-    local token_cache_key = kong.db.oauth2_tokens:cache_key(access_token)
+    local key = to_hex(sha1_bin(access_token))
+    local token_cache_key = kong.db.oauth2_tokens:cache_key(key)
```

Comment:
does this properly handle cache invalidations?

Reply:
Good question. I don't think so.  Any solutions?

Reply:
I think tokens are only inserted, and deleted, and selected. So perhaps it is fine for now?

Reply:
No it is not fine, I will write an invalidation handler.